### DISCUSSION TITLE
Asking fir permission

### DISCUSSION THREAD

**Discussion by wemsx** [KyleBing/apple-chip#5] [General]
Your project is awesome ！And I'm planning to rebuild obe for Qualcomm and Hisilicon. Can I use your project directly？ I'll keep the copyright part.

**Comment by KyleBing** [comment]
Go ahead.

**Reply by wemsx** [reply]
Thanks. I hope I won't  let you down！

**Reply by KyleBing** [reply]
哥们，你可以说中文的，我能听懂。

**Reply by wemsx** [reply]
哈哈🤗
